cl/citations/tests.py

```diff
@@ -269,265 +290,216 @@ def test_citation_resolution(self) -> None:
         """Tests whether different types of citations (i.e., full, short form,
         supra, id) resolve correctly to opinion matches.
         """
-        # fmt: off
-
         # Opinion fixture info:
         # pk=7 is mocked with name 'Foo v. Bar' and citation '1 U.S. 1'
         # pk=8 is mocked with name 'Qwerty v. Uiop' and citation '2 F.3d 2'
         # pk=9 is mocked with name 'Lorem v. Ipsum' and citation '1 U.S. 50'
         # pk=11 is mocked with name 'Abcdef v. Ipsum' and citation '1 U.S. 999'
+        opinion7 = Opinion.objects.get(pk=7)
```

Comment:
All these object definitions are just copied-and-pasted from the previous version of the tests. I just extracted them into variables because each "test pair" now needs to reference them multiple times.